garage/tf/algos/npo.py

```diff
@@ -308,12 +313,21 @@ def _build_policy_loss(self, i):
                         policy_dist_info_valid,
                         name="lr")
 
+                    ll = pol_dist.log_likelihood_sym(
+                        i.valid.action_var,
+                        policy_dist_info_valid,
+                        name="log_likelihood")
+
+                    vanilla = ll * adv_valid
                     surr_vanilla = lr * adv_valid
 
                 if self._pg_loss == PGLoss.VANILLA:
-                    # VPG, TRPO use the standard surrogate objective
+                    # VPG uses the vanilla objective
+                    surr_obj = tf.identity(vanilla, name="surr_obj")
```

Comment:
i think the variable `surr_obj` now needs to be renamed since it's not necessarily surrogate